File: CourceObjective-C/SwitchExample/SwitchExample/ViewController.m
//
//  ViewController.m
//  SwitchExample
//
//  Created by Palotec on 21/12/2017.
//  Copyright © 2017 Vanilsonvl. All rights reserved.
//

#import "ViewController.h"

@interface ViewController ()

@end

@implementation ViewController

- (void)viewDidLoad {
    [super viewDidLoad];
    
}

- (IBAction)onSwitchChange:(id)sender {
    UISwitch *s = (UISwitch *) sender;
    if (s.on == YES) {
        _emailLabel.hidden = NO;
        _emailTextField.hidden = NO;
    } else {
        _emailLabel.hidden = YES;
        _emailTextField.hidden = YES;
    }
}

- (IBAction)onClick:(id)sender {
    if (_emailSwitch.on == YES) {
        [_emailSwitch setOn:NO animated: YES];
        _emailLabel.hidden = YES;
        _emailTextField.hidden = YES;
    } else {
        [_emailSwitch setOn:YES animated: YES];
        _emailLabel.hidden = NO;
        _emailTextField.hidden = NO;
    }
}

- (void)didReceiveMemoryWarning {
    [super didReceiveMemoryWarning];
    // Dispose of any resources that can be recreated.
}


@end

File: Course/Alert/Alert/ViewController.swift
//
//  ViewController.swift
//  Alert
//
//  Created by Palotec on 10/07/17.
//  Copyright © 2017 Vanilsonvl. All rights reserved.
//

import UIKit

class ViewController: UIViewController {

    override func viewDidLoad() {
        super.viewDidLoad()
        // Do any additional setup after loading the view, typically from a nib.
    }

    override func didReceiveMemoryWarning() {
        super.didReceiveMemoryWarning()
        // Dispose of any resources that can be recreated.
    }

    @IBAction func btnAlert(_ sender: Any) {
        
        let alertController = UIAlertController(title: "Adicionar", message: "Deseja adicionar o contato?", preferredStyle: .actionSheet)
        let actionConfirmation = UIAlertAction(title: "Confirmar", style: .default, handler: nil)
        let actionCancel = UIAlertAction(title: "Cancelar", style: .cancel, handler: nil)
        alertController.addAction(actionConfirmation)
        alertController.addAction(actionCancel)
        present(alertController, animated: true, completion: nil)
        
        
    }

}

